scripts/prohibit-tab-character

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+self_dir=$(cd "$(dirname "$0")" &>/dev/null; pwd -P)
+source "$self_dir"/utils.sh
+
+# .gitconfig can be excluded since git config files are customarily indented with tabs rather than spaces
+if git --no-pager grep -In $'\t' -- ':(exclude)./.gitconfig'; then
```

Comment:
`$'<string>'` is similar to other "regular" strings (`"<string>"`, `'<string>'`), but additionally resolves some extra escape sequences (so `$'<string>'` is roughly equivalent to `$(echo -e '<string>')`)